app/lib/document_navigation.rb

```diff
@@ -4,6 +4,8 @@ class DocumentNavigation
   DOCUMENT_CONTROLLERS = {
     "ID" => Documents::IdsController,
     "Spouse ID" => Documents::SpouseIdsController,
+    "Primary SSN or ITIN" => Documents::PrimarySsnItinsController,
```

Comment:
As far as I can tell this is the first time we're introducing the copy "Primary" to something that the user would see (in the docs intro list)... Should we keep this as just "SSN or ITIN" to avoid the case where the user isn't filing joint and would be confused about it?

Reply:
ah so i thought about this - there's already an "SSN or ITIN" and it's for dependents. We could either: A) rename the existing document type and do a backfill, B) combine all of the SSN/ITIN pages (wouldn't allow us to know whose is whose) C) leave this with "Primary" and rename the primary ID page to have "Primary" also for consistency